site/gatsby-site/tailwind.config.js

```diff
@@ -58,6 +58,7 @@ module.exports = {
     extend: {
       screens: {
         '1300px': { min: '1300px' },
+        '3xl': { min: '1920px' },
```

Comment:
What's this for? Looks like it may have sitewide implications

Reply:
The largest [breakpoint](https://tailwindcss.com/docs/responsive-design) provided by Tailwind is 2xl at 1536px. If I set the grid of cards to 3 columns at 1536px, it looks too cramped. Whereas if just leave it at 2, the cards get too wide on a larger screen. 3xl follows the pattern, and 1920px seems like a natural place to set it. It's not overriding a default, so it won't have effects anywhere except here. I'd rather have it set here rather than a one-off because if we need a larger breakpoint that 1535px again we should use the same one.